web/clusters.go

```diff
@@ -275,31 +289,19 @@ type ClustersRow struct {
 
 type ClustersTable []*ClustersRow
 
-func NewClustersTable(clusters map[string]*cluster.Cluster, hosts map[string]hosts.HostList) ClustersTable {
+func NewClustersTable(clusters map[string]*cluster.Cluster) ClustersTable {
 	var clusterTable ClustersTable
 
 	for id, c := range clusters {
 		var health string
+		// TODO: Cost-optimized has multiple SIDs
 		var sids []string
 
-		hl, ok := hosts[id]
-
-		if ok && len(hl) > 0 {
-			health = hl.Health()
-			set := make(map[string]struct{})
-
-			for _, h := range hl {
-				ss, err := h.GetSAPSystems()
-				if err != nil {
-					continue
-				}
-
-				for _, s := range ss {
-					_, ok := set[s.SID]
-					if !ok {
-						set[s.SID] = struct{}{}
-						sids = append(sids, s.SID)
-
+		for _, r := range c.Cib.Configuration.Resources.Clones {
```

Comment:
I don't understand why you re-implemented this code again. Does not `getHanaSID` fill this purpose?

Reply:
sorry, something went wrong when squashing commits, fixed